File: 0136-single-number/0136-single-number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        unordered_map <int,int> maps;
        for(int i=0; i<nums.size() ; i++) {
            maps[nums[i]]++ ;
        }
        for ( auto map : maps) {
            if (map.second ==1) {
                return map.first;
            }
        }
        return -1;
    }
};

File: 0136-single-number/0136-single-number.java
class Solution {
    public int singleNumber(int[] nums) {
        HashMap<Integer,Integer> mp = new HashMap<>();

        for(int it : nums) {
            mp.put( it , mp.getOrDefault(it,0) +1  );
        }
        for(int key : mp.keySet()) {
            if( mp.get(key) == 1 ) {
                return key;
            }
        }
        return -1;
    }
}